ISA-XLSX.md

```diff
@@ -1,22 +1,65 @@
 # ISA-XLSX format
 
-For detail on ISA framework terminology, please read the [ISA Abstract Model specification](isamodel.md).
+For detail on ISA framework terminology, please read the [ISA Abstract Model specification](https://isa-specs.readthedocs.io/en/latest/isamodel.html).
 
 This document describes the ISA Abstract Model reference implementation specified in the ISA-XLSX format. The XLSX format uses the SpreadsheetML markup language and schema to represent a spreadsheet document. Conceptually, using the terminology of the Spreadsheet ML specification [ISO/IEC 29500-1](https://www.loc.gov/preservation/digital/formats/fdd/fdd000398.shtml#:~:text=The%20XLSX%20format%20uses%20the,a%20rectangular%20grid%20of%20cells.), the document comprises one or more worksheets in a workbook. Every worksheet MUST contain one table object storing the metadata. Comments or auxillary information MAY be stored alongside with table objects in a worksheet.
 
-Below we provide the schemas and the content rules for valid ISA-XLSX documents. 
+**Table of contents**
+
+- [ISA-XLSX format](#isa-xlsx-format)
+- [Top-level metadata sheets](#top-level-metadata-sheets)
+  - [Ontology Source Reference section](#ontology-source-reference-section)
+    - [ONTOLOGY SOURCE REFERENCE](#ontology-source-reference)
+      - [Example](#example)
+  - [INVESTIGATION section](#investigation-section)
+    - [INVESTIGATION](#investigation)
+      - [Example](#example-1)
+    - [INVESTIGATION PUBLICATIONS](#investigation-publications)
+      - [Example](#example-2)
```

Comment:
I do not like how this looks too much, the idea was giving direct links to the samples, but i think this looks weird. other opinions?

Reply:
Maybe use `**Example**` instead of `#### Example`?